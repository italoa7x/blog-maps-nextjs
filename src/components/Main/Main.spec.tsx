import { render, screen } from '@testing-library/react'
import Main from './index'

describe('testing Main component', () => {
  it('should be render <Main/> component', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /Main component/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
