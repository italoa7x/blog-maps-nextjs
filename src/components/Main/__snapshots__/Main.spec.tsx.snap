// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Main component should be render <Main/> component 1`] = `
<div>
  <h1>
    Main component
  </h1>
</div>
`;
